starter-code/css/main.css

```diff
@@ -6,3 +6,172 @@ Green: #00B172
 White: #FFF
 
 */
+
+*{
+    margin : 0;
+    padding: 0;
+}
+body {
+    font-family: Helvetica;
+    font-weight: lighter;
+    font-size: 20px; 
+}
+
+nav {
+    display: flex;
```

Comment:
by adding  
```
z-index: 1; 
```

it puts your nav on top. So when you scroll, everything goes under the nav